<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidget</class>
 <widget class="QWidget" name="ServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>83</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="buttonFile">
     <property name="text">
      <string>Choose File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Microsoft JhengHei</family>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>File Server</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>130</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="buttonSend">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Send File</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Message Box ^^^^^^</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
